package-lock.json

```diff
@@ -0,0 +1,451 @@
+{
```

Comment:
Are we doing `package-lock.json`?

Reply:
Probably nuke from this PR and see if there is a macro change we want to make here? Could be an issue on the core / cli repo.

Reply:
For example apps I'd say no, just so we keep the amount of required files to a minimum and devs don't assume the platform needs a package-lock.json to function.